akka-http-core/src/main/scala/akka/http/impl/engine/http2/Http2StreamHandling.scala

```diff
@@ -97,6 +97,9 @@ private[http2] trait Http2StreamHandling { self: GraphStageLogic with LogHelper
         }
     }
 
+  def activeStreamCount(): Int =
+    streamStates.size // FIXME is this good enough?
```

Comment:
```
// FIXME is this good enough?
```

I think it is good, yeah. If we introduce this method, I think we should replace other usages of `streamStates.size` with it (see `hasCapacityToCreateStreams` above, for example).